app/User.php

```diff
@@ -4,15 +4,18 @@
 
 use Carbon\Carbon;
 use Illuminate\Foundation\Auth\User as Authenticatable;
+use League\OAuth2\Client\Token\AccessToken;
+use App\Http\Controllers\VatsimOAuthController;
 use Illuminate\Notifications\Notifiable;
 use Laratrust\Traits\LaratrustUserTrait;
 
 class User extends Authenticatable {
     use Notifiable;
     use LaratrustUserTrait;
     protected $table = 'roster';
-    protected $fillable = ['id', 'fname', 'lname', 'email', 'rating_id', 'canTrain', 'visitor', 'status', 'loa', 'del', 'gnd', 'twr', 'app', 'ctr', 'train_pwr', 'monitor_pwr', 'opt', 'initials', 'added_to_facility', 'max','max_minor_del','max_minor_gnd','max_minor_twr','max_minor_app','twr_solo_fields','twr_solo_expires'];
+    protected $fillable = ['id', 'fname', 'lname', 'email', 'rating_id', 'canTrain', 'visitor', 'status', 'loa', 'del', 'gnd', 'twr', 'app', 'ctr', 'train_pwr', 'monitor_pwr', 'opt', 'initials', 'added_to_facility', 'max','max_minor_del','max_minor_gnd','max_minor_twr','max_minor_app','twr_solo_fields','twr_solo_expires','access_token','refresh_token','token_expires'];
     protected $secret = ['remember_token', 'password', 'json_token'];
+    protected $hidden = ['access_token', 'refresh_token', 'token_expires'];
```

Comment:
I'm not completely sure what the difference between secret and hidden is, but these probably should be combined into one array.

Also I'm not sure why password is there, that can probably be removed